helix-view/src/editor.rs

```diff
@@ -238,9 +238,28 @@ impl Editor {
                     self.documents[view.doc].selection(view.id).clone(),
```

Comment:
After the change to `current_ref`, this can now be simplified to 
```suggestion
                    doc.selection(view.id).clone(),
```

Reply:
It would be good to also move this jump definition into `if !remove_empty_scratch`. That way we're only constructing and cloning the selection if we actually need the jump. To do that we would need to use `view_mut!` twice, but that's cheaper than a clone